release notes/v0.55.0.md

```diff
@@ -0,0 +1,205 @@
+k6 `v0.55.0` is here 🎉! This release includes:
+
+- ⚠️ The deprecated StatsD output has been removed.
```

Comment:
Not sure if we need here emoji, keeping in mind that this already quite “urgent” section.

Reply:
I thought this added a sweet touch that attracted the eye to something users should not overlook. Feel free to remove it if you prefer without it 👍🏻

Reply:
Trying those out 🚀!